lib/index.coffee

```diff
@@ -14,9 +15,27 @@ dockerMtimeStream()
 		current_mtimes = layer_mtimes
 
 garbageCollect = (reclaimSpace) ->
-	dokcerImageTree()
-	.then(annotateTree.bine(null, current_mtimes))
+	dockerImageTree()
+	.then(annotateTree.bind(null, current_mtimes))
 	.then(lruSort)
 	.then (candidates) ->
 		# Remove candidates until we reach `reclaimSpace` bytes
-		console.log('foobar')
+		# Candidates is a list of images, with the least recently used
+		# first in the list
+
+		# Decide on the images to remove
+		# TODO: Find a lodash function for this!
+		size = 0
+		remove = []
+		for image in candidates
```

Comment:
Ah `takeWhile`! I was trying things like `reduce` et al but that makes more sense :)